lib/commands/certificate.js

```diff
@@ -410,6 +410,30 @@ commands.mobileInstallCertificate = async function mobileInstallCertificate (opt
   }
 };
 
+/**
+ * Removes installed certificates for real devices only
+ * if [py-ios-device](https://github.com/YueChen-C/py-ios-device) tool
+ * is available on the server machine.
+ *
+ * @property {!string} name - Name of certificate to remove
+ * @returns {Object} Returns status acknowledgment if successfully removed certificate or 'None' 
+ * @throws {Error} If attempting to remote certificates for simulated device or if py-ios-device
+ * is not installed
+ */
+commands.mobileRemoveCertificate = async function mobileRemoveCertificate (opts = {})  {
+  const {
+    name
+  } = opts;
+  if (!this.isRealDevice()) {
+    throw new errors.NotImplementedError('This extension is only supported on real devices');
+  }
+  const client = new Pyidevice(this.opts.udid);
+  if (await client.assertExists(false)) {
+    return await client.removeProfile(name);
+  }
+  throw new Error('pyidevice is not installed on your system, command cannot be executed.');
```

Comment:
simply pass `true` to assertExists API instead

Reply:
Updated. I also updated this same bit in `mobileListCertificates` so they're consistent

```
  await client.assertExists(true);
  return await client.removeProfile(name);
```